tools/cli.js

```diff
@@ -29,7 +29,14 @@ function complete(input, completions) {
   return completions.filter(c => c.startsWith(input));
 }
 
-function tryComplete(input, completer) {
+function tryComplete(input, completions) {
+  if (!completions) return input;
+  completions = complete(input, completions);
+  if (completions.length === 1) return completions[0];
+  return input;
+}
+
+function tryCompleter(input, completer) {
```

Comment:
I find it very confusing to have two functions that are completely distinct yet have names that resemble each other so close.